### PR TITLE
Changed URL for trading

### PR DESCRIPTION
**Pull request by Rishabh-kapoor**
Changed URL from testnet to Binance API server to make trades

### CODE REVIEW

**Comment by rodoufu** [comment]
It should close this #60 right?

File: binance_f/constant/system.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-    Uri = "wss://stream.binancefuture.com/ws"
-#    Uri = "wss://fstream.binance.com/ws"
+#    Uri = "wss://stream.binancefuture.com/ws"
+    Uri = "wss://fstream.binance.com/ws"
 
 class RestApiDefine:
-    Url = "https://testnet.binancefuture.com"
-#    Url = "https://fapi.binance.com"
+#     Url = "https://testnet.binancefuture.com"
+    Url = "https://fapi.binance.com"
```
Comment (rodoufu) [inline binance_f/constant/system.py:8]: It looks good to me.

**Review by rodoufu — APPROVED** [review]
I've just tested and it has worked to get the account information and account balance.

File: binance_f/constant/system.py (hunk 1 above)
Comment (Rishabh-kapoor) [reply binance_f/constant/system.py:8]: Thanks, it works perfectly with updated URL

**Comment by Rishabh-kapoor** [comment]
> It should close #60 right?

Yes it will

**Comment by rodoufu** [comment]
@Binance-docs @XavierXIEXIN @BinanceAPItest do you intend to merge this PR?

**Pull request merged**